media-gfx/hydrus/hydrus-448.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+PYTHON_REQ_USE="sqlite"
+
+inherit python-single-r1 desktop optfeature
+
+DESCRIPTION="A booru-like media organizer for the desktop"
+HOMEPAGE="https://hydrusnetwork.github.io/hydrus/ https://github.com/hydrusnetwork/hydrus"
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/hydrusnetwork/hydrus.git"
+else
+	SRC_URI="https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+	KEYWORDS="~amd64"
+fi
+
+# hydrus itself is WTFPL
+# icons included are CC-BY-2.5
+LICENSE="WTFPL-2 CC-BY-2.5"
+SLOT="0"
+IUSE="+mpv +ffmpeg +lz4 socks +cloudscraper charts test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/html5lib[${PYTHON_USEDEP}]
+		dev-python/lxml[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/psutil[${PYTHON_USEDEP}]
+		dev-python/pyopenssl[${PYTHON_USEDEP}]
+		dev-python/pyside2[widgets,gui,charts?,${PYTHON_USEDEP}]
+		dev-python/pyyaml[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+		dev-python/send2trash[${PYTHON_USEDEP}]
+		dev-python/service_identity[${PYTHON_USEDEP}]
+		dev-python/six[${PYTHON_USEDEP}]
+		dev-python/twisted[${PYTHON_USEDEP}]
+		media-libs/opencv[python,png,jpeg,${PYTHON_USEDEP}]
+
+		>=dev-python/QtPy-1.9.0-r4[pyside2,${PYTHON_USEDEP}]
+		dev-python/beautifulsoup:4[${PYTHON_USEDEP}]
+
+		cloudscraper? ( dev-python/cloudscraper[${PYTHON_USEDEP}] )
+		ffmpeg? ( media-video/ffmpeg )
+		lz4? ( dev-python/lz4[${PYTHON_USEDEP}] )
+		mpv? (
+			media-video/mpv[libmpv,${PYTHON_USEDEP}]
+			dev-python/python-mpv[${PYTHON_USEDEP}]
+		)
+		socks? (
+			|| ( dev-python/requests[socks5,${PYTHON_USEDEP}]
+				dev-python/PySocks[${PYTHON_USEDEP}] )
+		)
+	')
+"
```

Comment:
How is this block sorted? Could you add a descriptive comment about it on top of RDEPEND?

Reply:
They're grouped together by type (no specific requirement, slot/version requirement, specfic use flags) and then sorted, I'll add some comments.